threat_intel/virustotal.py

```diff
@@ -66,6 +117,45 @@ def get_domain_reports(self, domains):
         return all_responses
 
     @MultiRequest.error_handling
+    def get_url_distribution(self, params=None):
+        """Retrieves a live feed with the latest URLs submitted to VT.
+
+        Args:
+            resources: a dictionary with name and value for optional arguments
+        Returns:
+            A dict with the VT report.
+        """
+        params = params  or {}
+        all_responses = {}
+        api_name = 'virustotal-url-distribution'
+
+        response_chunks = self._request_reports(params.keys(), params.values(), 'url/distribution')
+        self._extract_response_chunks(all_responses, response_chunks, api_name)
+
+        return all_responses
+
+    @MultiRequest.error_handling
+    def get_file_distribution(self, params=None):
+        """Retrieves a live feed with the latest hashes submitted to VT.
+
+        Args:
+            params: a dictionary with name and values for optional arguments,
+            such as: before (timestampe), after (timestamp), reports (boolean),
+            limit (retrieve limit file items).
+            Example: 'reports': 'true'
+        Returns:
+            A dict with the VT report.
+        """
+        params = params or []
```

Comment:
That's a python gotcha: mutable objects should never be passed as default argument! https://pythonconquerstheuniverse.wordpress.com/category/python-gotchas/